cmd/logger.go

```diff
@@ -106,29 +96,13 @@ func (log *Logger) Printf(format string, args ...interface{}) {
 	}
 }
 
-// NewLogger - returns a new initialized logger.
-func NewLogger() *Logger {
-	logger := logrus.New()
-	logger.Out = ioutil.Discard
-	logger.Level = logrus.DebugLevel
-
-	l := &Logger{
-		logger:        logger,
-		consoleTarget: NewConsoleLogger(),
-	}
-
-	// Adds a console logger.
-	logger.Hooks.Add(l)
-
-	return l
-}
-
 func getSource() string {
```

Comment:
Move this function to `namespace-lock.go` (is only consumer).  This will ease out making logging as separate package.

Reply:
Done.